### PR TITLE
Added random emoji generator made using HTML, CSS and Javascript

### PR DESCRIPTION
**Pull request by urjabahad**

### CODE REVIEW

File: EmojiGenerator/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const btn=document.querySelector("#emoji");
+    const emojis = [
+  "😄", "😁", "😆", "😅", "😂", "🤣", "😊", "😇", "🙂", "🙃", "😉", "😌", "😍",
+  "🥰", "😘", "😗", "😙", "😚", "😋", "😛", "😝", "😜", "🤪", "🤨", "🧐", "🤓", "😎",  "🤩",
+  "🥳", "😏", "😒", "😞", "😔", "😟", "😕", "🙁", "☹️", "😣", "😖", "😫", "😩"
+];
+btn.addEventListener("mouseover",()=>{
+    btn.innerHTML=emojis[Math.floor(Math.random()*emojis.length)];
+})
+</script>
```
Comment (iamwatchdogs) [inline EmojiGenerator/index.html:44]: Hey @urjabahad, Please make a separate js file for this small piece of code.

```suggestion
<!-- All JS code will be within script.js" -->
<script src="script.js"></script>
```

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by urjabahad** [comment]
done changes. pls check.thank you

**Review by iamwatchdogs — APPROVED** [review]
Requested Changes have been made. Reviewed and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing, @urjabahad. Please check your contribution on [GitHub Pages](https://grow-with-open-source.github.io/Javascript-Projects/).

**Pull request merged**